src/main/java/org/minions/devfund/marcos/battleship/Ocean.java

```diff
@@ -0,0 +1,233 @@
+package org.minions.devfund.marcos.battleship;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+
+/**
+ * It is in charge to represent a Ocean and it is ships.
+ */
+public class Ocean {
+
+    private static final Map<ShipType, Integer> OCEAN_SHIPS = new HashMap<>();

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert this Map to an EnumMap. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1640)